php_phongo.c

```diff
@@ -81,6 +81,19 @@ ZEND_DECLARE_MODULE_GLOBALS(mongodb)
 #endif
 #endif
 
+php_phongo_server_description_type_map_t
+php_phongo_server_description_type_map[PHONGO_SERVER_DESCRIPTION_TYPES] = {
+	{ PHONGO_SERVER_UNKNOWN, "Unknown" },
+	{ PHONGO_SERVER_STANDALONE, "Standalone" },
```

Comment:
I didn't think the field order was important. Each element is a two-field struct with `type` and `name` fields. The array is technically a map from the enum integer to a string. We can technically use the array offset in lieu of storing the `type` field and make this an array of strings, but I figured the two-field struct was more readable.